tests/codegen_testcases/unused_variable_elimination.sol

```diff
@@ -0,0 +1,198 @@
+// RUN: --emit cfg
+
+contract c2 {
+    int public cd;
+    function sum(int32 a, int32 b) public pure returns (int32) {
+        return a+b;
+    }
+// BEGIN-CHECK: c2::function::doSomething
```

Comment:
Windows tests failed at this line. I am getting an intermittent error here. Do you have any clue on that?

Reply:
From the output it looks like the test did not produce output. Not sure what happened there, but it was probably spurious. Let's see how is goes next time.

Reply:
The problem appears to be the leading white spaces.